source/query/recent.txt

```diff
@@ -33,6 +41,26 @@ You can also hover over a query and perform the following actions:
    * - :icon-fa5:`trash-alt`
      - Remove the query from the :guilabel:`Recent` list.
 
+
+Query Autocomplete Menu 
+-----------------------
+
+If the query bar is empty, you can click the filter bar to open the 
+query autocomplete menu and view up to 50 of your most recent queries. To scroll 
+through your most recent queries, you can use the :kbd:`↑` and :kbd:`↓` arrow 
+keys. If a query is highlighted on the menu, it displays a pop-up panel that 
+shows the filter, project, and sort stages of the query. To select a query from 
+the autocomplete menu, press :kbd:`Enter` or click the query. 
```

Comment:
A user can click `Enter` or `Tab` to select the highlighted autocomplete option

Reply:
Updated!